Project.toml

```
@@ -0,0 +1,13 @@
+name = "DustExtinction"
+uuid = "fb44c06c-c62f-5397-83f5-69249e0a3c8e"
+license = "MIT"
+version = "0.3.1"
+
+[deps]
+FITSIO = "525bcba6-941b-5504-bd06-fd0dc1a4d2eb"
+
```

Please add a `compat` section:
```suggestion

[compat]
FITSIO = "≥ 0.13.0"
julia = "≥ 1.0.0"

```